czech/kapsy/nuz/bandaz/dvere/dvere.md

```diff
@@ -0,0 +1,3 @@
+Projdeš dveřmi azjistiš že je tam hromada maršmeloidních humanoidu kteří vypadají jako panaček mischelin obraceny naruby 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.